arlas-core/src/main/java/io/arlas/server/core/services/CollectionReferenceService.java

```diff
@@ -428,52 +444,57 @@ public void checkIfIndexAllowedForOrganisations(CollectionReference collection,
             return;
         }
         // In case of using arlas IAM
-        if(policyEnforcer.get().equals("io.arlas.filter.impl.HTTPPolicyEnforcer")){
+        if (policyEnforcer.get().equals("io.arlas.filter.impl.HTTPPolicyEnforcer")) {
             String indexName = collection.params.indexName;
-            if(!indexName.contains("@")){
+            if (!indexName.contains("@")) {
                 throw new InvalidParameterException("Index name must begin with org_name@.");
             }
-            long validOrgNbr = Arrays.stream(organisations.get().split(",")).filter(o -> indexName.split("@")[0].equals(o) ).count();
-            if(validOrgNbr == 0){
+            long validOrgNbr = Arrays.stream(organisations.get().split(",")).filter(o -> indexName.split("@")[0].equals(o)).count();
+            if (validOrgNbr == 0) {
                 throw new NotAllowedException("You are not authorized to create this collection.");
             }
         }
     }
 
     public Boolean checkIfAllowedForOrganisations(CollectionReference collection,
-                                               Optional<String> organisations,
-                                               boolean ownerOnly)
+            Optional<String> organisations,
+            boolean ownerOnly)
             throws CollectionUnavailableException {
-        if (organisations.isEmpty()) {
-            // no header, we'll trust the column filter if any
-            LOGGER.debug("No organisation header");
-            return true;
-        }
+        if (configuration.arlasCheckOrganisations) {
```

Comment:
why don't you start by:
```
if not arlasCheckOrganisations:
   return true
```

This would simplify the reading, don't you think?

Reply:
Also, it is important to make sure that if arlasCheckOrganisations is null, then it is like arlasCheckOrganisations=true !!!